web/sites/default/config/field.field.group.elected_official.field_enable_body_content.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 88c62d96-9a67-423c-b6e0-060c12ed38c8
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.group.field_enable_body_content
+    - group.type.elected_official
+  module:
+    - tmgmt_content
+third_party_settings:
+  tmgmt_content:
+    excluded: false
+id: group.elected_official.field_enable_body_content
+field_name: field_enable_body_content
+entity_type: group
+bundle: elected_official
+label: 'Enable Body Content'
```

Comment:
Can you change the label and the description below to match the "Custom Menu" display name for field_body_content? It took me a minute to realize that the "Custom Menu" field is enabled by this. :) It may cause confusions among the group editors.

Reply:
Done.